README.md

```diff
@@ -41,10 +41,28 @@ in Internet Explorer on Windows 7 for instance (because of the wrong MIME type).
 
 JavaScript Cookie supports [npm](https://www.npmjs.com/package/js-cookie) and [Bower](http://bower.io/search/?q=js-cookie) under the name `js-cookie`.
 
+- #### Bower
+  ```
+  bower install js-cookie --save
```

Comment:
Are you sure we need to document the command? Bower exist for a while so I expect people to know the command of how to install it as long as we document it's supported. We can't teach people how to use their tools, they should know what they are doing before using the library.